internal/explorer/eplorer_utils_test.go

```diff
@@ -0,0 +1,361 @@
+						Description: "hey this is a bool, required!",
+					}),
+				},
+				{
+					Name: "float64_prop",
+					In:   "query",
+					Schema: base.CreateSchemaProxy(&base.Schema{
+						Type:        []string{"number"},
+						Format:      "float",
+						Description: "hey this is a float64!",
+					}),
+				},
+			},
+			commonParams: []*high.Parameter{
+				{
+					Name:        "string_prop",
+					Required:    pointer(false),
+					In:          "path",
+					Description: "hey this is a string, required and overidden!",
+					Schema: base.CreateSchemaProxy(&base.Schema{
+						Type:        []string{"string"},
+						Format:      util.OAS_format_password,
+						Description: "you shouldn't see this because the description is overridden!",
+					}),
+				},
+			},
+			want: []*high.Parameter{
+				{
+					Name:        "string_prop",
+					Required:    pointer(true),
+					In:          "path",
+					Description: "hey this is a string, required and overidden!",
+					Schema: base.CreateSchemaProxy(&base.Schema{
+						Type:        []string{"string"},
+						Format:      util.OAS_format_password,
+						Description: "you shouldn't see this because the description is overridden!",
+					}),
+				},
+				{
+					Name:     "bool_prop",
+					Required: pointer(true),
+					In:       "query",
+					Schema: base.CreateSchemaProxy(&base.Schema{
+						Type:        []string{"boolean"},
+						Description: "hey this is a bool, required!",
+					}),
+				},
+				{
+					Name: "float64_prop",
+					In:   "query",
+					Schema: base.CreateSchemaProxy(&base.Schema{
+						Type:        []string{"number"},
+						Format:      "float",
+						Description: "hey this is a float64!",
+					}),
+				},
+			},
+		},
+		"no read op": {
+			noReadOp:   true,
+			readParams: nil,
+			commonParams: []*high.Parameter{
+				{
+					Name:        "string_prop",
+					Required:    pointer(false),
+					In:          "path",
+					Description: "hey this is a string, required and overidden!",
+					Schema: base.CreateSchemaProxy(&base.Schema{
+						Type:        []string{"string"},
+						Format:      util.OAS_format_password,
+						Description: "you shouldn't see this because the description is overridden!",
+					}),
+				},
+			},
+			want: []*high.Parameter{
+				{
+					Name:        "string_prop",
+					Required:    pointer(false),
+					In:          "path",
+					Description: "hey this is a string, required and overidden!",
+					Schema: base.CreateSchemaProxy(&base.Schema{
+						Type:        []string{"string"},
+						Format:      util.OAS_format_password,
+						Description: "you shouldn't see this because the description is overridden!",
+					}),
+				},
+			},
+		},
+	}
+	for name, testCase := range testCases {
+		name, testCase := name, testCase
+		t.Run(name, func(t *testing.T) {
+			t.Parallel()
+
+			var readOp *high.Operation = nil
+			if !testCase.noReadOp {
+				readOp = &high.Operation{
+					Parameters: testCase.readParams,
+				}
+			}
```

Comment:
Rather then having two separate test case variables with this boolean `noReadOp`, can we just define `readOp` in the test case itself?

Example test cases:
```go
		"nil read parameters": {
			readOp: &high.Operation{
				Parameters: nil,
			},
			commonParams: []*high.Parameter{
				{
					Name:        "common_string_prop",
					Required:    pointer(true),
					In:          "path",
					Description: "hey this is a string, required and overidden!",
					Schema: base.CreateSchemaProxy(&base.Schema{
						Type:        []string{"string"},
						Format:      util.OAS_format_password,
						Description: "you shouldn't see this because the description is overridden!",
					}),
				},
			},
			want: []*high.Parameter{
				{
					Name:        "common_string_prop",
					Required:    pointer(true),
					In:          "path",
					Description: "hey this is a string, required and overidden!",
					Schema: base.CreateSchemaProxy(&base.Schema{
						Type:        []string{"string"},
						Format:      util.OAS_format_password,
						Description: "you shouldn't see this because the description is overridden!",
					}),
				},
			},
		},
		"no read op": {
			readOp: nil
			commonParams: []*high.Parameter{
				{
					Name:        "string_prop",
					Required:    pointer(false),
					In:          "path",
					Description: "hey this is a string, required and overidden!",
					Schema: base.CreateSchemaProxy(&base.Schema{
						Type:        []string{"string"},
						Format:      util.OAS_format_password,
						Description: "you shouldn't see this because the description is overridden!",
					}),
				},
			},
			want: []*high.Parameter{
				{
					Name:        "string_prop",
					Required:    pointer(false),
					In:          "path",
					Description: "hey this is a string, required and overidden!",
					Schema: base.CreateSchemaProxy(&base.Schema{
						Type:        []string{"string"},
						Format:      util.OAS_format_password,
						Description: "you shouldn't see this because the description is overridden!",
					}),
				},
			},
		},
```